docker/docker-compose.yml

```diff
@@ -610,3 +614,54 @@ services:
     command: "./wait-for-it.sh mariadb:3306 --timeout=720 --strict -- ./start.sh"
     profiles:
       - bitcoin-mempool
+
+  prometheus:
+    image: prom/prometheus:v2.45.2
+    restart: always
+    volumes:
+      - ./local-prometheus.yml:/etc/prometheus/prometheus.yml
+      # - prometheus-data:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
+    ports:
+      - 9090:9090
+    depends_on:
+      - vector
+    profiles:
+      - observability
+
+  grafana:
+    image: grafana/grafana:10.2.3-ubuntu
```

Comment:
nit: 11.4.0

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/1105/commits/28a04360ff6f6208c8732ce53802bca21b45af7e.